src/main/java/exceptions/ItemNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class ItemNotFoundException extends Exception {
```

Comment:
make both `ItemNotFoundException` and `MemberNotFoundException` share a common parent, they are very similar